microprofile.jdt/com.redhat.microprofile.jdt.core/schema/javaFeatureParticipants.exsd

```diff
@@ -51,6 +52,50 @@
       </complexType>
    </element>
 
+   <element name="codeAction">
+      <annotation>
+         <documentation>
+            Java codeAction participant.
+         </documentation>
+      </annotation>
+      <complexType>
+         <attribute name="kind" use="default" value="quickfix">
+            <annotation>
+               <documentation>
+                  The code action kind top category. This kind is used to filter code action according the &apos;only&apos; information of LSP CodeActionContext (see https://microsoft.github.io/language-server-protocol/specification#textDocument_codeAction).
+               </documentation>
+            </annotation>
+            <simpleType>
+               <restriction base="string">
+                  <enumeration value="quickfix">
+                  </enumeration>
+                  <enumeration value="refactor">
+                  </enumeration>
+                  <enumeration value="source">
+                  </enumeration>
+               </restriction>
+            </simpleType>
+         </attribute>
+         <attribute name="targetDiagnostic" type="string">
+            <annotation>
+               <documentation>
+                  The target diagnostic where the code action must be applied. This target can use diagnostic code (ex: &apos;ImplementHealthCheck&apos;) or concat of diagnostic source, &apos;#&apos; and diagnostic code (ex:  : &apos;microprofile-health#ImplementHealthCheck&quot;)
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="class" type="string" use="required">
+            <annotation>
+               <documentation>
+                  Name of a class that implements .IJavaCodeActionParticipant

```

Comment:
Please remove the period in ".IJavaCodeActionParticipant"

Reply:
fixed